packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx

```diff
@@ -146,20 +154,18 @@ export const RecordInlineCellContainer = ({
       )}
       <StyledValueContainer>
         <RecordInlineCellValue
-          {...{
-            displayModeContent,
-            customEditHotkeyScope,
-            disableHoverEffect,
-            editModeContent,
-            editModeContentOnly,
-            isDisplayModeFixHeight,
-            buttonIcon,
-            label,
-            loading,
-            readonly,
-            showLabel,
-            isCentered,
-          }}
+          displayModeContent={displayModeContent}
```

Comment:
Nice, could you try avoid drilling down in RecordInlineCellValue and call those props where they are need at the lowest level possible ? 

We should avoid calling RecordInlineCellContext in the FieldDisplay and FieldInput hierarchy, as they are called both by RecordInlineCell and RecordTableCell

Reply:
Thank you for the feedback! I updated the scripts to avoid drilling down into RecordInlineCellValue in RecordInlineCellContainer and getting the context directly from RecordInlineCellValue. Please let me know any further changes are required!